README.md

```diff
@@ -12,67 +12,54 @@ Applications using broadside employ a configuration file that looks something li
 ```ruby
 Broadside.configure do |config|
   config.application = 'hello_world'
-  config.docker_image = 'lumoslabs/hello_world'
+  config.docker_image = 'docker.io/lumoslabs/hello_world'
```

Comment:
without a domain in the name docker will assume the image is stored in dockerhub, but we're using quay so specifying that domain before the project is necessary